### DISCUSSION TITLE
Possibility to discard audio only of selected segments

### DISCUSSION THREAD

**Discussion by SILENT-da** [mifi/lossless-cut#743] [Ideas]
Hello,
as said in title, it would be nice to have the possibility to mute a segment. This could be useful in different ways, i.e. you cut a sequence with 'Keyframe cut', (and because of the keyframe) the new file starts a little bit earlier than you need, so you make this litte part at the beginning a new muted segment in a second step. Another example, you mute different parts of a video for an alternative audio overlay in a further step.
Thanks! :-)

**Comment by sasha2048** [comment]
Probably related: https://github.com/mifi/lossless-cut/issues/90
